src/main/java/dekk/pw/pokemate/tasks/CatchPokemon.java

```diff
@@ -55,7 +55,7 @@ public void run() {
                 if (target != null && pokeball != null) {
                     Walking.setLocation(context);
                     EncounterResult encounterResult = target.encounterPokemon();
-                    if (encounterResult.wasSuccessful()) {
+                    if (encounterResult.wasSuccessful() && !Config.getIgnoreCatchingPokemon().contains(target.getPokemonId().getNumber())) {
```

Comment:
instead of pokemon.get(0), it should be `pokemon.stream.filter(!Config.getIgnoreCatchingPokemon().contains(target.getPokemonId().getNumber())`